include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -66,26 +258,1047 @@ class HighPressureGasTransport : public MultiTransport
+     * pure species.
+     *
+     * The primary equation is given by Equation 1 in @cite ely-hanley1981 :
+     *
+     * @f[
+     *   \eta_i(\rho,T) = \eta_0(\rho_0, T_0) F_{\eta}
+     * @f]
+     *
+     * Where @f$ F_{\eta} @f$ is defined by Equation 2 in @cite ely-hanley1981 :
+     *
+     * @f[
+     *   F_{\eta} = \left( \frac{MW_i}{MW_0} \right)^{1/2} f_{i,0}^{1/2} h_{i,0}^{-2/3}
+     * @f]
+     *
+     * The `0` subscripts refer to the reference fluid, which is methane in this case,
+     * and the `i` subscripts refer to the species of interest. No mixing rules
+     * need to be used here because this is for a pure species. As such, the terms
+     * @f$ f_{i,0} @f$ and @f$ h_{i,0} @f$ have simpler expressions. The value of
+     * @f$ f_{i,0} @f$ is given by Equation 7 in @cite ely-hanley1981 :
+     *
+     * @f[
+     *   f_{i,0} = \frac{T_c^i}{T_c^0} \theta_i(T_R^i, V_R^i, \omega_i)
+     * @f]
+     *
+     * and the value of @f$ h_{i,0} @f$ is given by Equation 8 in @cite ely-hanley1981 :
+     *
+     * @f[
+     *  h_{i,0} = \frac{V_c^i}{V_c^0} \phi_i(T_R^i, V_R^i, Z_{c,i}, \omega_i)
+     * @f]
+     *
+     * Where @f$ \theta_i @f$ and @f$ \phi_i @f$ are shape factors of Leach and Leland
+     * ( see thetaShapeFactor() and phiShapeFactor() ), @f$ T_c^i @f$ is the critical
+     * temperature of species i, @f$ T_c^0 @f$ is the critical temperature of the
+     * reference fluid, @f$ V_c^i @f$ is the critical volume of species i,
+     * @f$ V_c^0 @f$ is the critical volume of the reference fluid, @f$ Z_{c,i} @f$
+     * is the critical compressibility of species i, and @f$ \omega_i @f$
+     * is the acentric factor of species i.
+     *
+     * @param V Molar volume of the species
+     * @param Tc Critical temperature of the species
+     * @param Vc Critical volume of the species
+     * @param Zc Critical compressibility of the species
+     * @param acentric_factor Acentric factor of the species
+     * @param mw Molecular weight of the species
+     */
+    double elyHanleyDilutePureSpeciesViscosity(double V, double Tc, double Vc,
+                                               double Zc, double acentric_factor,
+                                               double mw);
+
+
+    /**
+     * Returns the theta shape factor of Leach and Leland for a pure species.
+     *
+     * The shape factor @f$ \theta_i @f$ is defined by Equation 11 in @cite ely-hanley1981
+     * as follows:
+     *
+     * @f[
+     *   \theta_i(T_R^i, V_R^i, \omega_i) = 1 + (\omega_i - \omega_0)F(T_R^i, V_R^i)]
+     * @f]
+     *
+     * Where @f$ \omega_i @f$ is the acentric factor of species i, @f$ \omega_0 @f$ is
+     * the acentric factor of the reference fluid, and @f$ F(T_R^i, V_R^i) @f$ is a
+     * function of the reduced temperature and reduced volume of species i. The
+     * function @f$ F(T_R^i, V_R^i) @f$ is defined by Equation 13 in
+     * @cite ely-hanley1981:
```

Comment:
Doxygen appears to struggle with this, looking for a citation named `ely-hanley1981:`, so I guess you need to write:
```suggestion
     * @cite ely-hanley1981 :
```
here and in a couple other places below.